[core]/es_extended/server/main.lua

```diff
@@ -385,6 +387,16 @@ AddEventHandler('playerDropped', function(reason)
 	end
 end)
 
+AddEventHandler("esx:playerLoaded", function(playerId, xPlayer, isNew)
+    local job = xPlayer.getJob().name
+    ESX.JobsPlayerCount[job] = (ESX.JobsPlayerCount[job] or 0) +1
+end)
+
+AddEventHandler("esx:setJob", function(src, job, lastJob)
+    ESX.JobsPlayerCount[lastJob.name] = (ESX.JobsPlayerCount[lastJob.name] or 1) - 1
```

Comment:
Handle please minus values.

Reply:
I'm not sure I understand what you mean.

Reply:
It tracks any users job, so how could someone still have a police job if the count is 0?

Reply:
I have pushed a change anyways.